internal/e2e/rollup_code_splitting/BUILD.bazel

```diff
@@ -6,8 +6,8 @@ rollup_bundle(
         ["*.js"],
         exclude = ["*.spec.js"],
     ),
-    additional_entry_points = ["internal/e2e/rollup_code_splitting/additional_entry.js"],
-    entry_point = "internal/e2e/rollup_code_splitting/main1.js",
+    additional_entry_points = ["build_bazel_rules_nodejs/internal/e2e/rollup_code_splitting/additional_entry"],
```

Comment:
do we have to change addl_entry_points at the same time?

Reply:
nope. not sure how that change got in. backed it out. the `.js` is optional there but no reason to change it.